src/Makefile

```diff
@@ -79,9 +80,21 @@ $(OUTPUT)/retsnoop.skel.h: $(OUTPUT)/mass_attach.bpf.o
 $(OUTPUT)/retsnoop.o: $(OUTPUT)/retsnoop.skel.h $(OUTPUT)/calib_feat.skel.h
 $(OUTPUT)/mass_attacher.o: $(OUTPUT)/retsnoop.skel.h $(OUTPUT)/calib_feat.skel.h
 
-$(OUTPUT)/addr2line.embed.o: ../tools/addr2line
+../sidecar/target/release/addr2line::
+	$(call msg,CARGO,addr2line)
+	$(Q)if [ -e $@ ]; then touch $@.timestamp; fi
+	$(Q)cargo build --manifest-path=../sidecar/Cargo.toml --release
+	$(Q)if [ ! -e $@.timestamp -o $@ -nt $@.timestamp ]; then \
```

Comment:
oh, this timestamp stuff is pretty ugly :( how about we copy a file in addr2line.embed.o target locally and strip that one?.. See below.